<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myDataDialog</class>
 <widget class="QDialog" name="myDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="deviceIDLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>21</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="timestampLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="birdNameLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
